src/transformers/models/encoder_decoder/configuration_encoder_decoder.py

```diff
@@ -43,15 +43,15 @@ class EncoderDecoderConfig(PretrainedConfig):
     Examples:
 
     ```python
-    >>> from transformers import BertConfig, EncoderDecoderConfig, EncoderDecoderModel
```

Comment:
this line should not be changed, keep it as `BertConfig, EncoderDecoderConfig, EncoderDecoderModel`